src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -172,6 +175,25 @@ internal static class DefinitionItemExtensions
             return properties;
         }
 
+        private static ImmutableArray<AdditionalProperty> GetDisplayableProperties(ISymbol definition)
+        {
+            var displayableProperties = ImmutableArray<AdditionalProperty>.Empty;
```

Comment:
I think we can still share the logic in this method (going from `ISymbol -> ImmutableArray<AdditionalProperty>`) with the method doing the same transformation for references. I can do this as a follow-up change as well.